PeerTests/ConnectorAPI/PRListAPITests.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Peer.ConnectorAPI.Tests
+{
+    public class PRListAPITests
+    {
+        // TODO: Finish unit test
```

Comment:
Good TODO :) lets focus on that before expanding to azdo

Reply:
Just started :) Gonna kill it today.